desktop/php/jMQTT_eqpt.php

```diff
@@ -125,57 +125,18 @@
-					<option value="boiteauxlettres">{{Boite aux Lettres}}</option>
-					<option value="water-boiler">{{Chaudière}}</option>
-					<option value="chauffage">{{Chauffage}}</option>
-					<option value="molecule-co">{{CO}}</option>
-					<option value="compteur">{{Compteur}}</option>
-					<option value="contact">{{Contact}}</option>
-					<option value="feuille">{{Culture}}</option>
-					<option value="custom">{{Custom}}</option>
-					<option value="dimmer">{{Dimmer}}</option>
-					<option value="energie">{{Energie}}</option>
-					<option value="window-closed-variant">{{Fenêtre}}</option>
-					<option value="garage">{{Garage}}</option>
-					<option value="humidity">{{Humidité}}</option>
-					<option value="humiditytemp">{{Humidité et Température}}</option>
-					<option value="hydro">{{Hydrométrie}}</option>
-					<option value="fire">{{Incendie}}</option>
-					<option value="ir2">{{Infra Rouge}}</option>
-					<option value="home-flood">{{Inondation}}</option>
-					<option value="jauge">{{Jauge}}</option>
-					<option value="lightbulb">{{Lumière}}</option>
-					<option value="light">{{Luminosité}}</option>
-					<option value="meteo">{{Météo}}</option>
-					<option value="motion">{{Mouvement}}</option>
-					<option value="multisensor">{{Multisensor}}</option>
-					<option value="nab">{{Nabaztag}}</option>
-					<option value="gate">{{Portail}}</option>
-					<option value="door">{{Porte}}</option>
-					<option value="prise">{{Prise}}</option>
-					<option value="motion-sensor">{{Présence}}</option>
-					<option value="power-plug">{{Prise de courant}}</option>
-					<option value="radiator">{{Radiateur}}</option>
-					<option value="relay">{{Relais}}</option>
-					<option value="433">{{RF433}}</option>
-					<option value="rfid">{{RFID}}</option>
-					<option value="sms">{{SMS}}</option>
-					<option value="bell">{{Sonnerie}}</option>
-					<option value="remote">{{Télécommande}}</option>
-					<option value="teleinfo">{{Téléinfo}}</option>
-					<option value="tv">{{Télévison}}</option>
-					<option value="temp">{{Température}}</option>
-					<option value="thermostat">{{Thermostat}}</option>
-					<option value="fan">{{Ventilation}}</option>
-					<option value="volet">{{Volet}}</option>
-					<option value="wifi">{{Wifi}}</option>
-					<option value="zigbee">{{Zigbee}}</option>
-					<option value="zwave">{{ZWave}}</option>
-				</select>
+				<select class="form-control eqLogicAttr" data-l1key="configuration" data-l2key="icone"></select>
+				<script>
+					// Initialize Icon dropbox from jmqttIcons global var
+					// This MUST be done here, only once and before loading plugin.template.js
+					var icos = $('.eqLogicAttr[data-l1key=configuration][data-l2key=icone]');
+					icos.html('');
+					$.each(jmqttIcons, function(key) {
+						opt = new Option(jmqttIcons[key]['name'], jmqttIcons[key]['id']);
+						opt.setAttribute('file', jmqttIcons[key]['file']);
+						icos.append(opt);
+					});
```

Comment:
What about icon names translation?

Reply:
Yes, that's the point of this change, it is done server side here:
https://github.com/Domochip/jMQTT/blob/330d3a1cfefacc4edce75cc9270f5a8241a94841/desktop/php/jMQTT.php#L63-L70